core/API/DataTableManipulator/ReportTotalsCalculator.php

```diff
@@ -155,7 +155,7 @@ protected function manipulateDataTable($dataTable)
         }
 
         if (isset($totalRow)) {
-            $totals = $row->getColumns();
+            $totals = $totalRow->getColumns();
```

Comment:
This worked correctly with `$row` as well, as the foreach loop above does a `break` after assigning `$totalRow`. So `$row` remains with the same value as `$totalRow`.